autotest/alg/contour.py

```diff
@@ -103,37 +103,30 @@ def contour_1():
 
     lyr = ogr_ds.ExecuteSQL("select * from contour order by elev asc")
 
-    if lyr.GetFeatureCount() != len(expected_envelopes):
-        print('Got %d features. Expected %d' % (lyr.GetFeatureCount(), len(expected_envelopes)))
-        return 'fail'
+    assert lyr.GetFeatureCount() == len(expected_envelopes), ('Got %d features. Expected %d' % (lyr.GetFeatureCount(), len(expected_envelopes)))
```

Comment:
General comment: Can you avoid massively long lines?  It's especially difficult in github side by side diffs where the break depends on html rendering rather than a logical place

Reply:
Related @schwehr I've rarely found assert's argument part necessary when using pytest: pytest's default is to show you the difference between the left and right hand sides in the expression. Dropping that will shorten lines.

Reply:
Actually, it might be worth putting a little effort into this. So far I've put zero effort in :)